src/components/account/DeleteAccount.tsx

```diff
@@ -43,6 +43,13 @@ const DeleteAccount = () => {
           <div className="error">{validationMessage}</div>
         )}
       </div>
+
+      <div className="STASHTEST">
```

Comment:
I think this may have accidentally been pushed

Reply:
Whoops! Did not realize that